### PR TITLE
Initial CONTRIBUTING and CODE-OF-CONDUCT

### PR DESCRIPTION
**Pull request by kaungst**
This provides an initial formal mechanism to propose updates the spec.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+  future option to loosen when motivated by a real use case.
+
+- **Understandability is just as important as correctness**
+
+  The HMIS API specification, despite describing technical behavior, is intended to be
+  read by people. Use natural tone and include motivation and examples.
+
+## RFC Contribution Champions
+
+Contributing to GraphQL requires a lot of dedicated work. To set clear
```
Comment (HUD-Data-Lab) [inline CONTRIBUTING.md:98]: I was under the impression that we are working with a JSON REST API format and not a GraphQL. Confirming that you meant GraphQL and not REST API.

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others’ private information, such as a physical or electronic address, without explicit permission
+
+## Our Responsibilities
+
+HMIS API Specification contributors are responsible for clarifying the standards of positive behavior.
+
+The Department of Housing and Urban Development (HUD) has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions.
```
Comment (HUD-Data-Lab) [inline CODE-OF-CONDUCT.md:28]:
```suggestion
The Department of Housing and Urban Development (HUD) and the HUD Data Lab has the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions.  
```

Comment (HUD-Data-Lab) [reply CODE-OF-CONDUCT.md:28]: Requesting that we include HUD Data Lab explicitly in this as well. The HUD Data Lab (ICF) is the party that will be responsible for most of the maintaining of this repository and everything in it.

**Review by HUD-Data-Lab — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kaungst) [reply CONTRIBUTING.md]: dun

File: CODE-OF-CONDUCT.md (hunk 2 above)
Comment (kaungst) [reply CODE-OF-CONDUCT.md]: dun

**Review by HUD-Data-Lab — APPROVED** [review]

**Pull request merged**